optuna/progress_bar.py

```diff
@@ -75,15 +75,25 @@ def _init_valid(self) -> None:
         optuna_logging.disable_default_handler()
         optuna_logging._get_library_root_logger().addHandler(_tqdm_handler)
 
-    def update(self, elapsed_seconds: float) -> None:
+    def update(self, elapsed_seconds: float, best_trial: int, best_value: Optional[float]) -> None:
```

Comment:
To avoid the error due to the multi-objective, how about taking `study`?
```suggestion
    def update(self, elapsed_seconds: float, study: Study) -> None:
```

Reply:
Note that we need to import `study` as follows to avoid circular import.

```python
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from optuna.study import Study
```

Reply:
Applied :)